src/ameba/rule/lint/bad_directive.cr

```diff
@@ -48,7 +48,9 @@ module Ameba::Rule::Lint
 
     private def check_rules(source, token, rules)
       bad_names = rules - ALL_RULE_NAMES - ALL_GROUP_NAMES
-      issue_for token, "Such rules do not exist: %s" % bad_names.join(", ") if bad_names.any?
+      unless bad_names.empty?
+        issue_for token, "Such rules do not exist: %s" % bad_names.join(", ")
+      end
```

Comment:
What the purpose of this change? I would stick to the previous approach if you don't mind :)

If this is about `.any?` vs `.emtpy?` performance, please change it to one-liner `unless bad_names.empty?`

Reply:
I've changed it to `unless bad_names.empty?` as requested.